/**:
  ros__parameters:
    joystick_surge_gain: 60.0
    joystick_sway_gain: 60.0
    joystick_heave_gain: 17.5
    joystick_roll_gain: 30.0
    joystick_pitch_gain: 20.0
    joystick_yaw_gain: 25.0

    guidance_surge_gain: 0.0002
    guidance_sway_gain: 0.0002
    guidance_heave_gain: 0.0002
    guidance_roll_gain: 0.0002
    guidance_pitch_gain: 0.0002
    guidance_yaw_gain: 0.0002

    debounce_duration: 0.25
